Wikipedia/Code/TalkPageEmptyView.swift

```diff
@@ -41,6 +41,7 @@ final class TalkPageEmptyView: SetupView {
         scrollView.translatesAutoresizingMaskIntoConstraints = false
         scrollView.contentInsetAdjustmentBehavior = .never
         scrollView.automaticallyAdjustsScrollIndicatorInsets = false
+        scrollView.showsVerticalScrollIndicator = true
```

Comment:
The documentation says this property defaults to yes, so I suspect setting this here might not do anything. Something you can try is flashing the scroll view indicators whenever the scroll view appears (I'm trying it here in `layoutSubviews` so that it also flashes upon rotation):

```
    override func layoutSubviews() {
        super.layoutSubviews()
        
        if scrollView.bounces {
            scrollView.flashScrollIndicators()
        }
    }
```